reference-implementation/to-upstream-wpts/transform-stream/general.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+}
+
+test(() => {
+  const ts = new TransformStream({ transform() { } });
+
+  assert_true(Object.getOwnPropertyDescriptor(Object.getPrototypeOf(ts), 'writeable'), 'it has a writable property');
```

Comment:
This existing test of the property descriptor is kind of bad. Maybe replace it with something like https://github.com/w3c/web-platform-tests/blob/a838bae2db982f96ddb6760ca88e699d4514884e/streams/readable-streams/general.js#L56-L61 instead.